functions/system.bash

```diff
@@ -46,7 +46,7 @@ needed_packages() {
 
 timezone_setting() {
   # shellcheck source=/etc/openhabian.conf disable=SC1091
-  source "$CONFIGFILE"
+  source "$configfile"
```

Comment:
I think those calls can be removed. if configfile is defined, it was already sourced... 
Also with my proposed change, it should have been anyway from the calling bash script.

Reply:
ok